### ISSUE TITLE
document is not defined on React Native

### ISSUE THREAD

**Issue by danieljgp2** [cpunion/react-actioncable-provider#24]
Hello im currently working on a React Native application and i'm getting this error.

<img width="336" alt="Captura de pantalla 2019-03-23 a la(s) 23 12 59" src="https://user-images.githubusercontent.com/24395686/54874003-70eb3380-4dc1-11e9-849b-ee97a7742c98.png">

I already try 
```
import ActionCable from 'actioncable';
const cable = ActionCable.createConsumer('ws://test.example.com/cable');
<ActionCableProvider cable={cable}>...</ActionCableProvider>;
```
like the documentation says [https://github.com/cpunion/react-native-actioncable#use-with-react-actioncable-provider](url) but i still getting the same error.

I'm currently using react-native-actioncable but likes the documentation says, is now unmaintained

**Comment by Rana-Muhammad-Amir** [comment]
@danieljgp2 Any update regarding this....? i am also facing this issue on react-native 
@cpunion please guide

**Comment by danieljgp2** [comment]
@Rana-Muhammad-Amir No, i was using the deprecated version cause was the only one that works fro me but i quit from this library.

**Comment by amorimlucas** [comment]
@danieljgp2 Did you find a better library?

**Comment by abdulwahabcs** [comment]
@danieljgp2  Anything you found please share here that would be helpfull

**Comment by danieljgp2** [comment]
@amorimlucas @abdulwahabcs im not using ActionCable anymore, im using FCM as background messages to handle any realtime event, you can take this as an approach otherwise use a pub/sub
